pt_miniscreen/pages/network/ap.py

```diff
@@ -9,16 +9,16 @@ class Page(PageBase):
     def __init__(self, size):
         row_data = NetworkPageData(
             first_row=RowDataText(
-                icon_path="sys_info/networking/antenna.png",
-                text=lambda: get_ap_mode_status().get("ssid", ""),
+                icon_path="sys_info/networking/antenna-small.png",
+                text=lambda: get_ap_mode_status().get("ssid", "Not active"),
             ),
             second_row=RowDataText(
-                icon_path="sys_info/networking/padlock.png",
+                icon_path="sys_info/networking/padlock-small.png",
                 text=lambda: get_ap_mode_status().get("passphrase", ""),
             ),
             third_row=RowDataText(
-                icon_path="sys_info/networking/home.png",
+                icon_path="sys_info/networking/home-small.png",
                 text=lambda: get_ap_mode_status().get("ip_address", ""),
             ),
         )
-        super().__init__(size=size, row_data=row_data)
+        super().__init__(size=size, row_data=row_data, title="Access Point")
```

Comment:
I think 'Wi-Fi Hotspot' or maybe even just 'Hotspot' if that doesn't fit is better. people are more likely to get it imo and I think that's what we call it in onboarding now

Reply:
actually it does say 'Access Point' on the settings page, so if we change one it should be both

Reply:
Yeah I agree 'Wi-Fi Hotspot' is a better name. I've updated it in both places :)